packages/react-instantsearch-dom/src/hooks.js

```diff
@@ -0,0 +1,16 @@
+import { useContext } from 'react';
+import { InstantSearchContext } from 'react-instantsearch-core';
+
+export function useSearch() {
+  const { store, widgetsManager } = useContext(InstantSearchContext);
+  const getResults = () => store.getState().results;
+  const searchBox = widgetsManager
+    .getWidgets()
+    .find(widget => widget.name.startsWith('AlgoliaSearchBox'));
+  const setQuery = query => searchBox && searchBox.refine(query);
```

Comment:
I wonder if there should be a setSearchState where you can add query yourself; this doesn't use multi-index yet I think

Reply:
Yeah, we should expose `searchState` and `setSearchState`. This was just to see if it works :)